style.css

```diff
@@ -0,0 +1,557 @@
+.body {
+  max-width: 1920px;
+  min-width: 320px;
+  margin: 0 auto;
+  background: url('pic/iswanto-arif-448003-unsplash.png') no-repeat;
+  background-size: cover;
+  overflow: visible;
+  height: 100vh;
+}
+
+.header {
+  width: 100%;
+  padding: 87px 114px 0 199px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0 0 130px 0;
+}
+
+.header::after {
+  content: "";
+  position: absolute;
+  height: 100vh;
+  width: 100vw;
+  background-color: #fff;
+  top: -100vh;
+  left: 0;
+  z-index: 2;
+  transition: .3s ease;
+}
+
+.container {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin-right: 4px;
+  letter-spacing: .25pt;
+}
+
+.nav__link:not(:last-child) {
+  margin: 0 35px 0 0;
+}
+
+.nav__link {
+  color: #fff;
+  text-decoration: none;
+  font-size: 22px;
+  line-height: 26px;
+  font-family: 'Museo', sans-serif;
+  border-bottom: 1px solid transparent;
+  transition: .3s ease;
+}
+
+.nav__link:hover {
+  border-bottom: 1px solid #fff;
+  opacity: 0.8;
+}
+
+.nav__toggle {
+  display: none;
+}
+
+.nav__burder {
+  display: none;
+}
+
+.bali {
+  display: flex;
+  flex-direction: column;
+  padding-left: 140px;
+}
+
+.bali-wrapper {
+  position: relative;
+}
+
+.bali__title {
+  display: flex;
+  font-family: 'PTSerif', sans-serif;
+  font-style: normal;
+  margin: 0;
+  font-weight: bold;
+  font-size: 280px;
+  color: #f8eee4;
+  opacity: 0.4;
+}
+
+.bali__subtitle {
+  font-family: 'PTSerif', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 98px;
+  margin: 0 0 9px 0;
+  color: #f8eee4;
+  position: absolute;
+  left: 83px;
+  bottom: 11px;
+}
+
+.bali__text {
+  font-family: 'Museo', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 26px;
+  color: #f8eee4;
+  margin: 0 0 38px 10px;
+  max-width: 551px;
+  padding-left: 15px;
+}
+
+.picture {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 115px;
+}
+
+.picture__pic:not(:last-child) {
+  margin: 0 32px 0 0;
+}
+
+.picture__pic {
+  padding-bottom: 3px;
+}
+
+.picture__pic_active {
+  padding-bottom: 0;
+}
+
+.picture__pic_cut {
+  margin-right: 0;
+}
+
+.picture__pic_tablet {
+  display: none;
+}
+
+.bali__btn {
+  width: 292px;
+  height: 61px;
+  background: #d4bea3;
+  border-radius: 48px;
+  border: none;
+  font-weight: bold;
+  font-size: 22px;
+  line-height: 26px;
+  color: #fff;
+  cursor: pointer;
+  transition: .5s;
+}
+
+.bali__btn:hover {
+  background: none;
+  border: 1px solid #d4bea3;
+}
+
+#cut {
+  margin-right: 0;
+}
+
+@media screen and (min-width: 1920px) {
+  

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)